hybris/configure.ac

```diff
@@ -117,8 +117,12 @@ AC_SUBST(DEFAULT_HYBRIS_LD_LIBRARY_PATH)
 AC_ARG_WITH(android-headers,
   [  --with-android-headers=DIR     Use android headers available in DIR. See utils/extract-headers.sh ],
   [
-    AM_CONDITIONAL([HAS_LIBNFC_NXP_HEADERS], [test -f $withval/libnfc-nxp/phLibNfc.h])
-    AS_IF([test -f $withval/libnfc-nxp/phLibNfc.h], [ANDROID_HEADERS_CFLAGS="-I$withval -I$withval/libnfc-nxp"],[ANDROID_HEADERS_CFLAGS="-I$withval"])
+    CPPFLAGS_saved="$CPPFLAGS"
+    CPPFLAGS="$CPPFLAGS -I$withval -I$withval/libnfc-nxp"
+    AC_CHECK_HEADER([phLibNfc.h], [ANDROID_HEADERS_CFLAGS="-I$withval -I$withval/libnfc-nxp"], [ANDROID_HEADERS_CFLAGS="-I$withval"])
+    CPPFLAGS="$CPPFLAGS_saved"
+
+    AM_CONDITIONAL([HAS_ANDROID_LIBNFC_NXP], [test -n "$ac_cv_header_phLibNfc_h"])
     AC_SUBST([ANDROID_HEADERS_CFLAGS])
   ],
   [ PKG_CHECK_MODULES(ANDROID_HEADERS, android-headers,, exit) ]
```

Comment:
You need to define HAS_ANDROID_LIBNFC_NXP here as well as otherwise the generated makefiles will complain about the symbol HAS_ANDROID_LIBNFC_NXP being missing.